app/screens/global_threads/threads_list/index.ts

```diff
@@ -32,7 +32,7 @@ const enhanced = withObservables(['tab', 'teamId'], ({database, tab, teamId}: Pr
         threads: teamThreadsSyncObserver.pipe(
             switchMap((teamThreadsSync) => {
                 const earliest = tab === 'all' ? teamThreadsSync?.[0]?.earliest : 0;
-                return queryThreadsInTeam(database, teamId, getOnlyUnreads, false, true, true, earliest).observe();
+                return queryThreadsInTeam(database, teamId, getOnlyUnreads, true, true, true, earliest).observe();
```

Comment:
Do you mind explaining why changing hasReplies to true would improve thread handling?

You'd get posts to appear in the thread list only when reply_count > 0?.  I followed a single post with no reply and it's not showing in my thread list.  Is adding this make a difference?

Again curious how it improves.

Reply:
The test case I know for this is:
- Create a post
- Follow it
- Create a reply
- Delete the reply
- Go to Threads

You will see a thread without responses